postman-collection/Positions.postman_collection.json

```diff
@@ -0,0 +1,105 @@
+{
+	"info": {
+		"_postman_id": "192c3934-d74c-4bd1-bcfb-e9284a2a041a",
```

Comment:
why did you create a new Postman Schema instead of using the existing one?

Reply:
My bad, added to the existing file.